tests/components/sensor/test_openhardwaremonitor.py

```diff
@@ -0,0 +1,52 @@
+"""The tests for the Open Hardware Monitor platform."""
+import unittest
+import requests_mock
+from homeassistant.components.sensor import openhardwaremonitor
+from homeassistant.setup import setup_component
+
+from tests.common import load_fixture, get_test_home_assistant
+
+
+class TestOpenHardwareMonitorSetup(unittest.TestCase):
+    """Test the Open Hardware Monitor platform."""
+
+    def add_entities(self, new_entities, update_before_add=False):
```

Comment:
Not sure where you copied this from, but you don't need this. Instead, just read it from the state machine.